File: 7KYU/Minimize Sum Of Array (Array Series #1)/solution.rb
def min_sum(arr)
    asc = arr.sort_by{|a| a}
    dsc = arr.sort_by{|a| -a}
    i = 0
    res = 0
    while i < arr.length() do
      res+= asc[i] * dsc[i]
      i=i+1
    end 
    return res/2
  end

File: 7KYU/Minimize Sum Of Array (Array Series #1)/solution.cpp
#include <vector>
using namespace std;

int minSum (vector<int>passed)
{
  sort(passed.begin(), passed.end());
  int size = passed.size() - 1;
  int p = 0;
  for(int i = 0; i <= size/2; i++)
  {
     p = p + (passed[i] * passed[size-i]);
  }
  return p; 
}